docs/commands/graph.query.md

```diff
@@ -902,33 +902,35 @@ This section contains information on all supported functions from the Cypher que
 |stDevP(_expr_)                       | Returns the population standard deviation over a set of numeric values. null values are ignored <br> Returns null when _expr_ has no evaluations                       |
-|sum(_expr_)                          | Returns the sum of a set of numeric values. null values are ignored <br> Returns 0 when _expr_ has no evaluations                                                      |
+|sum(_expr_)                          | Returns the sum of a set of numeric values. null values are ignored <br> Returns 0 when _expr_ has no evaluations                                                         |
 
 ## List functions
 
 | Function                             | Description|
 | ------------------------------------ | :----------|
-| head(_expr_)                         | Returns the first element of a list <br> Returns null when _expr_ evaluates to null or an empty list                                                                                                 |
+| head(_expr_)                         | Returns the first element of a list <br> Returns null when _expr_ evaluates to null or an empty list                                                                                                    |
 | keys(_expr_)                         | Returns a list of strings: all key names for given map or all property names for a given node or edge <br> Returns null when _expr_ evaluates to null                                                |
-| last(_expr_)                         | Returns the last element of a list <br> Returns null when _expr_ evaluates to null or an empty list                                                                                                  |
+| last(_expr_)                         | Returns the last element of a list <br> Returns null when _expr_ evaluates to null or an empty list                                     
+| list.dedup(_list_) * | Given a list, returns a similar list after removing duplicate elements <br> Order is preserved, duplicates are removed from the end of the list <br> Returns null when _list_ evaluates to null <br> Error when _list_ does not evaluate to a list or to null |
+| list.insert(_list_, _idx_, _val_[, _dups_ = TRUE]) * | Given a list, returns a list after inserting a given value at a given index <br> _idx_ is 0-based when non-negative, or from the end of the list when negative <br> Returns null when _list_ evaluates to null <br> Returns _list_ when _val_ evaluates to null <br> Returns _list_ when _idx_ evaluates to an integer not in [-NumItems-1 .. NumItems] <br> When _dups_ evaluats to FALSE: returns _list_ when _val_ evaluates to a value that is already an element of _list_  <br> Error when _list_ does not evaluate to a list or to null <br> Error when _idx_, if specified, does not evaluate to an integer <br> Error when _dups_, if specified, does not evaluate to a Boolean |
```

Comment:
"Returns _list_ when _idx_ evaluates to an integer not in [-NumItems-1 .. NumItems]"
consider rephrasing:
1. Returns _list_ when _idx_ is out of range
2. Returns _list_ when _idx_ is out of bounds


"returns _list_ when _val_ evaluates to a value that is already an element of _list_"
consider rephrasing:
returns _list_ when _val_ is in _list_

"Error when _idx_, if specified, does not evaluate to an integer "
`__idx__` is mandatory it must be specified.

Reply:
1st remark: I prefer it this way - to indicate that _idx_ can also be negative
2nd remark: I prefer it this way - e.g., `5+5` evaluates to 10 which is already an element.
3rd remark: fixed